inventory/group_vars/uk.yml

```diff
@@ -15,4 +15,12 @@ timezone: London
 developer_email: dev.ofnuk@gmail.com
 
 users_sysadmin:
-  - pmackay
+  - lindhop
+  - enricostn
```

Comment:
I see that you add Matt's SSH public key in `/files` but you don't add him in this list.